app/Console/Commands/AllocateStandForArrival.php

```diff
@@ -25,9 +25,13 @@ public function handle()
             $this->info('Skipping arrival stand allocation');
             return;
         }
-
+        $allAircraft = NetworkAircraft::all();
```

Comment:
Is it worth filtering the list of aircraft down to those with EG as a prefix ICAO to reduce the iteration needed?

Reply:
c7c5e32

Reply:
Looks great :)